addOns/pscanrules/CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Handle an IllegalArgumentException that could occur in the CSP scan rule if multiple CSP headers were present and one (or more) had a report-uri directive when trying to merge them.
 - Allow to ignore cookies in same site and loosely scoped scan rules.
 - The Application Error scan rule will not alert on web assembly responses.
+- The Suspicious Comments scan rule will include the offending line as evidence.
```

Comment:
@psiinon I referred to this change

Reply:
But not the fact that the rule now raises alerts for each instance? I think that should be mentioned too.

Reply:
Sure, fixed :)